python_examples/basic_infection_history_simulation/simulation_flow.py

```diff
@@ -49,7 +49,8 @@
 inf_history_params = {"output_dir": os.path.join(os.path.dirname(__file__),
                                                  "simulation_outputs"),
                       "status_output": True,
-                      "infectiousness_output": True}
+                      "infectiousness_output": True,
+                      "compress": False}  # Set to True if compression wanted
```

Comment:
```suggestion
                      "compress": False}  # Set to True if compression desired
```
Tiny thing but this wording seems more standard (but I'm really sorry if this takes you over the line limit and fails flake8